roland/Objects/Page.swift

```diff
@@ -49,9 +49,9 @@ class Page {
         var foundEndOfFrontMatter = false
         while !foundEndOfFrontMatter {
             if lines.count == 0 {
-                foundEndOfFrontMatter = true
```

Comment:
What was the bug that this fixed?

Reply:
this prevented an array index out of bounds error on line 55